src/main/java/pingis/controllers/MiscController.java

```diff
@@ -0,0 +1,18 @@
+package pingis.controllers;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+public class MiscController {
+
+    @RequestMapping("/")
+    public String index() {
+        return "index";
+    }
+
+    @RequestMapping("/sandbox")
```

Comment:
Given the MiscController only has the `/index` route, I'd rename it to `IndexController` as @DeepIntuition said. Adding a "whatever goes" type of controller isn't particularly good design, and I'd rather not add one now when it's not clear if we'll actually need one. Add it *when* the need arises.